src/viking.js

```diff
@@ -1,14 +1,116 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(health, strength) {
+    this.health = health;
+    this.strength = strength;
+  }
+
+  attack() {
+    return this.strength
+
+  }
+
+  receiveDamage(damage) {
+    this.health =- damage;
+  }
```

Comment:
Little but important syntax mistake!  Should be ` this.health -= damage;`

To avoid this , do not hesitate and make it more obvious when you're not sure why it does not work. For example just right it like this and you won't be wrong:
 `this.health = this.health - damage;`